models/db.py

```diff
@@ -77,11 +78,12 @@
 )
 
 if db(db.courses.id > 0).isempty():
-    db.courses.insert(course_name='boguscourse', term_start_date=datetime.date(2000, 1, 1)) # should be id 1
-    db.courses.insert(course_name='thinkcspy', base_course = 'thinkcspy', term_start_date=datetime.date(2000, 1, 1))
-    db.courses.insert(course_name='pythonds', base_course = 'pythonds', term_start_date=datetime.date(2000, 1, 1))
-    db.courses.insert(course_name='overview', term_start_date=datetime.date(2000, 1, 1))
-    db.courses.insert(course_name='publicpy3', base_course='pip2', term_start_date=datetime.date(2000, 1, 1))
+    db.courses.insert(id=1, course_name='boguscourse', term_start_date=datetime.date(2000, 1, 1)) # should be id 1, to avoid error in auto populate of auth_users, where course_id defaults to 1
+    db.courses.insert(id=2, course_name='thinkcspy', base_course = 'thinkcspy', term_start_date=datetime.date(2000, 1, 1))
+    db.courses.insert(id=3, course_name='pythonds', base_course = 'pythonds', term_start_date=datetime.date(2000, 1, 1))
+    db.courses.insert(id=4, course_name='overview', term_start_date=datetime.date(2000, 1, 1))
+    db.courses.insert(id=5, course_name='publicpy3', base_course='pip2', term_start_date=datetime.date(2000, 1, 1))
```

Comment:
These probably don't belong in the general commit.   Also hardcoding the id's is going to get courses_id_seq out of sync.  If the database is empty then these courses will insert with the ids you want anyway.  So I can't imagine why you would need to hardcode them.